Lib/idlelib/idle_test/test_help_about.py

```diff
@@ -170,5 +176,37 @@ def test_file_display(self):
                 self.assertEqual(self.view.called, True)
 
 
+class BitnessTest(unittest.TestCase):
+    "Test helper function for bitness of system."
+
+    def test_darwin(self):
+        help_about.sys = mock.MagicMock()
+        platform = mock.PropertyMock(return_value='darwin')
+        size = mock.PropertyMock(return_value=2**32)
+        type(help_about.sys).platform = platform
+        type(help_about.sys).maxsize = size
+        help_about.architecture = mock.Mock(return_value=('64bit', 'ELF'))
+        self.assertEqual(help_about._bitness(), '32')
+        platform.assert_called_with()
+        size.assert_called_with()
+
+        size = mock.PropertyMock(return_value=2**64)
+        type(help_about.sys).maxsize = size
+        self.assertEqual(help_about._bitness(), '64')
+        platform.assert_called_with()
+        size.assert_called_with()
+
+    def test_not_darwin(self):
+        help_about.sys = mock.MagicMock()
+        platform = mock.PropertyMock(return_value='linux')
+        size = mock.PropertyMock(return_value=2**32)
+        type(help_about.sys).platform = platform
+        type(help_about.sys).maxsize = size
+        help_about.architecture = mock.Mock(return_value=('64bit', 'ELF'))
+        self.assertEqual(help_about._bitness(), '64')
+        platform.assert_called_with()
+        size.assert_not_called()
```

Comment:
I have decided that these tests are way too much for something that almost does not need to be tested.  Testing that the implementation is what we can see it is by reading the code is not very useful, though a common mistake.  If the implementation is wrong, it is wrong.  Reduce this to testing what we know must be true if there is no bug.
`# untested ;-)`
def test_build_bits(self)
    self.assertIn(help_about.build_bits, (32, 64))

Put is some current class.  Rename if needed.